src/app/components/pages/send-skycoin/send-verify/send-verify.component.ts

```diff
@@ -0,0 +1,52 @@
+import { Component, EventEmitter, Input, Output, ViewChild } from '@angular/core';
+import { MatSnackBar, MatSnackBarConfig } from '@angular/material';
+
+import { PreviewTransaction } from '../../../../app.datatypes';
+import { WalletService } from '../../../../services/wallet.service';
+import { ButtonComponent } from '../../../layout/button/button.component';
+
+@Component({
+  selector: 'app-send-verify',
+  templateUrl: './send-verify.component.html',
+  styleUrls: ['./send-verify.component.scss'],
+})
+export class SendVerifyComponent {
+  @ViewChild('sendButton') sendButton: ButtonComponent;
+  @ViewChild('backButton') backButton: ButtonComponent;
+  @Input() transaction: PreviewTransaction;
+  @Output() onBack = new EventEmitter<boolean>();
+
+  constructor(
+    private walletService: WalletService,
+    private snackbar: MatSnackBar
+  ) {}
+
+  send() {
+    this.snackbar.dismiss();
+    this.sendButton.resetState();
+    this.sendButton.setLoading();
+
+    this.walletService.injectTransaction(this.transaction.inputs, this.transaction.outputs)
+      .subscribe(
+        () => this.onSuccess(),
+        (error) => this.onError(error)
+      );
+  }
+
+  back() {
+    this.onBack.emit(false);
+  }
+
+  private onSuccess() {
+    this.sendButton.setSuccess();
+    this.walletService.loadBalances();
+    this.onBack.emit(true);
```

Comment:
Please, do not return immediately, but after a moment, as in the desktop wallet. That small amount of time, with `sendButton.setSuccess()`, is currently being used as a method for the user to know that the transaction was sended.

Also, refresh the balance just after the transaction is sent.

Reply:
Fixed.